src/util/FileSystem.ts

```diff
@@ -80,7 +83,7 @@ export default class FileSystem {
     }
 
     private getFileName(fullPath: string): string {
-        const lastSlashIndex: number = fullPath.lastIndexOf("/");
-        return fullPath.substring(lastSlashIndex + 1);
+        const firstSlashIndex: number = fullPath.indexOf("/");
+        return fullPath.substring(firstSlashIndex + 1);
```

Comment:
What does this change do?

Reply:
This change returns the whole path rather than just the filename, e.g. ```src/resolution/DNS_Resolver.java``` instead of ```DNS_Resolver.java```

Reason why I needed to do this is github diffs have the whole path as the ```fileChanged```'s name so this allows me to check if a method is contained in a diff/file easier

Reply:
@kiyomih 

Does this affect your use of `getFileName` for the dependency graph generator?

Reply:
Nope, it doesn't!

Reply:
Might require a tiny change on my end, but it's no biggie